backend/src/main/kotlin/tools/vitruv/vitruvAdapter/vitruv/impl/exception/DisplayViewException.kt

```diff
@@ -0,0 +1,7 @@
+package tools.vitruv.vitruvAdapter.vitruv.impl.exception
```

Comment:
imo it makes more sense to move the exception to the top level exception package to group all exceptions together